cylc/flow/cfgspec/globalcfg.py

```diff
@@ -99,6 +99,53 @@
         'sort order': [VDR.V_STRING, 'definition', 'alphanumeric'],
     },
 
+    # job platforms
+    'job platforms': {
+        '__MANY__': {
+            'batch system': [VDR.V_STRING, 'background'],
+            'batch submit command template': [VDR.V_STRING],
```

Comment:
Added here in place of the current task runtime setting. I can't see any reason why this would need to be set for an individual task. Longer term we should look to whether this setting is needed - if it is there are probably other similar commands we need to be able to configure.

Reply:
(I've sometimes found it useful for testing, e.g. to fake the "submitted" state for 30 secs with background jobs).

Reply:
I think in that use case I'd tell you to define a platform in your user config.